social-network/src/AuthorBio.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './AuthorBio.css'
+
+class AuthorBio extends React.Component {
+
+  static propTypes = {
+    author: PropTypes.shape({
+      name: PropTypes.string.isRequired,
+      image: PropTypes.string.isRequired,
+      isMediumMember: PropTypes.bool.isRequired
+    }),
+  }
+
+  indicateMediumMember (flag) {
+    if (flag) {
+      return <h3>_____</h3>
+    } else {
+      return null
+    }
+  }
+
+  render() {
+    const authorBioStyle = {
+      backgroundColor: "purple",
+      width: "80%"
+    }
+ 
+    // const authImageStyle = {
+    //   width: "5px",
+    //   height: "5px"
+    //   // float: "left"
+    // }
+    
+    return (
+      <div style={authorBioStyle}>
+        {this.indicateMediumMember(this.props.author.isMediumMember)}
```

Comment:
You could also consider rendering this inline with a ternary or && statement : 
{this.props.author.isMediumMember && <h3>_____</h3>}